tests/modules/nf-core/deepvariant/main.nf

```diff
@@ -35,21 +35,6 @@ workflow test_deepvariant_cram_intervals {
     DEEPVARIANT_INTERVALS ( cram_tuple_ch, fasta, fai, [] )
 }
 
-workflow test_deepvariant_no_fasta_gzi {
```

Comment:
Why remove this test?

Reply:
Because it gives this error in the new version:
```
           ValueError: NOT_FOUND: could not load fasta and/or fai for fasta genome.fasta.gz                                                                                                    
           parallel: This job failed:                                                                                                                                                          
           /opt/deepvariant/bin/make_examples --mode calling --ref genome.fasta.gz --reads test.paired_end.sorted.bam --examples ./make_examples.tfrecord@2.gz --channels insert_size --gvcf   
         ./gvcf.tfrecord@2.gz --regions chr22:0-40001 --task 1  
```